src/main/java/com/example/onlinestore/user/User.java

```diff
@@ -0,0 +1,66 @@
+package com.example.onlinestore.user;
+
+import java.util.Arrays;
+import java.util.Collection;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.validation.constraints.Email;
+
+import org.springframework.security.core.GrantedAuthority;
+import org.springframework.security.core.authority.SimpleGrantedAuthority;
+import org.springframework.security.core.userdetails.UserDetails;
+
+import lombok.Data;
+
+@Data
+@Entity
+public class User implements UserDetails {
+
+    private static final long serialVersionUID = 1L;
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.AUTO)
+    private Long id;
+
+    @Column(name = "user_name")
+    private String userName;
+    private String password;
+    private String name;
+    @Email
+    private String email;        
+
+    @Override
+    public Collection<? extends GrantedAuthority> getAuthorities() {
+        return Arrays.asList(new SimpleGrantedAuthority("USER"));
```

Comment:
[IDE] Please pay more attention to [your IDE warnings](https://www.jetbrains.com/help/idea/code-inspection.html). This code shows:

> Call to 'asList()' with only one argument 

```suggestion
        return Collections.singletonList(new SimpleGrantedAuthority("USER"));
```

Reply:
will do :+1: